crates/biome_js_analyze/src/lint/nursery/use_strict_mode.rs

```diff
@@ -91,22 +95,62 @@ impl Rule for UseStrictMode {
             PreferredQuote::Double => "\"use strict\"",
             PreferredQuote::Single => "'use strict'",
         };
-        let value = JsSyntaxToken::new_detached(JsSyntaxKind::JSX_STRING_LITERAL, value, [], []);
-        let use_strict_diretcive = make::js_directive(value)
-            .with_semicolon_token(make::token(T![;]))
-            .build();
-        let directives = make::js_directive_list(
+
+        // check if the first statement has a newline as the first trivia.
+        let is_statement_first_trivia_newline = node
+            .statements()
+            .syntax()
+            .first_leading_trivia()
+            .map(|f| {
+                f.pieces()
+                    .next()
+                    .map_or(false, |piece| piece.kind() == TriviaPieceKind::Newline)
```

Comment:
You can use https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some_and

Reply:
fixed